CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.37.2
+
+This release includes [changes and fixes](https://github.com/getsentry/rrweb/releases/tag/1.102.0) around text masking in Replay's `rrweb` dependency.
```

Comment:
Might be helpful to just embed rrweb changelog in here?

```suggestion
This release includes [changes and fixes](https://github.com/getsentry/rrweb/releases/tag/1.102.0) around text masking in Replay's `rrweb` dependency:
  - feat: Add `maskAllText` option
  - feat: With maskAllText, mask the attributes: placeholder, title, `aria-label`
  - feat: fix masking on `textarea`
```

Reply:
Sure, done: [f811483](https://github.com/getsentry/sentry-javascript/pull/7158/commits/f8114836e7ca7b2e812ea0711a448a47be2048d9)